### DISCUSSION TITLE
div tag of top html not fullwidth  on mobile

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#25659] [Help] [answered]
Hi, why my div tag didn't fullwidth on mobile. I don't understand why :( Everything fine on PC
![image](https://user-images.githubusercontent.com/15695252/120239664-f499e800-c288-11eb-9cc1-6d77772515f1.png)

Thank you for any help

**Comment by IRediTOTO** [comment]
The body tag not full to
![image](https://user-images.githubusercontent.com/15695252/120239772-388ced00-c289-11eb-9795-f0c5fbe82930.png)

**Comment by wwwdepot** [comment, marked as answer]
It could be something overflowing outside of the viewport
